src/plugin.ts

```diff
@@ -88,7 +88,6 @@ export class Plugin {
                 ReflectionKind.ClassOrInterface |
                     ReflectionKind.Enum |
                     ReflectionKind.Function |
-                    ReflectionKind.ObjectLiteral |
```

Comment:
This was removed by TypeDoc as never used https://github.com/TypeStrong/typedoc/releases/tag/v0.25.0

> Removed ReflectionKind.ObjectLiteral that was never used by TypeDoc.